### ISSUE TITLE
Issue if Windows taskbar is on the side of screen.

### ISSUE THREAD

**Issue by kadonnelly13** [amitmerchant1990/pomolectron#1]
I installed the tool but I have my taskbar on the side of the screen which is causing issues.  The window is unable to be moved to the outside of the taskbar.

![image](https://cloud.githubusercontent.com/assets/22972794/24003982/431c672e-0a22-11e7-9ae4-cd5977da173e.png)

**Comment by amitmerchant1990** [comment]
Hi @kadonnelly13 Thanks for trying out Pomolectron. 

I'll try to fix this particular issue and will update the app accordingly.

**Comment by wadbr** [comment]
Yep having this issue too, is there a way to edit where the window appears when clicked?

**Comment by afino** [comment]
Hey there, same issue here, looks like it still exists. Taskbar is on the left and the Pomolectron window appears off screen and can't be moved. I'm running Win7 x64.

![pomolectron_bug](https://user-images.githubusercontent.com/23667131/57043202-32218680-6c67-11e9-9b2e-7bdd62a198d2.png)
